jenkins/osa_onmetal.functions.groovy

```diff
@@ -347,12 +347,15 @@ def start_api_uptime_tests() {
 
     String host_ip = get_onmetal_ip()
 
-    // run the API uptime tests
+    // run the API uptime tests.  Note that the output directory goes up 2
+    // levels.  That is done to ensure that the output file goes into the root
+    // for this project, rather than buried in the api_uptime/api_uptime
+    // subdirectory.
```

Comment:
I think its more flexible this way - we are now only dependent on OUR filesystem structure, not the OS.  Plus I believe this is how @lamarwhitej does it with the "during" tests (he will correct me of I'm wrong).